### ISSUE TITLE
Colors are not right in vim

### ISSUE THREAD

**Issue by redbmk** [imcatnoone/toothpaste#42]
I came across this colorscheme the other day and really liked the screenshots of it, but it looks very, very different when I use it in vim. Normally I have a transparent, gray background, but with this colorscheme I get a solid, dark blue background (solid turquoise for areas that have no code). I'm running vim 7.4 in linux.
![screenshot from 2016-06-05 10-41-56](https://cloud.githubusercontent.com/assets/83964/15804482/26ba494a-2b0a-11e6-9fa9-993872b75f0b.png)

**Comment by redbmk** [comment]
vimdiff also looks pretty terrible. I'm thinking maybe this looks ok in gvim, but not in regular vim...

![screenshot from 2016-06-05 11-00-39](https://cloud.githubusercontent.com/assets/83964/15804541/de68e7a2-2b0c-11e6-9fdb-011014177ce0.png)

**Comment by imcatnoone** [comment]
@redbmk whoa! that should **not** be the case. Sorry about that. Pinging @jondot on this as he re-created the theme for VIM. Want to chime in here?

**Comment by jondot** [comment]
Hi there, I'm not sure I experienced that, it might be that I'm using gvim, and for shell on iTerm, I usually have to set up base colors that are good for the color scheme I'm using (see [here](https://github.com/chriskempson/base16-iterm2)). I'm not sure - but maybe this relates to 256 vs 16 colors on terminal? does that make sense?

**Comment by redbmk** [comment]
Yeah, that makes sense, but I'm using 256 colors. I've played around with it a little bit, but I'm not good at picking what colors go good together (I can just tell that these don't look right). For a few spots I've just removed the background color altogether (just for term) and it looks fine. For vimdiff I just don't know what colors to pick.

I'm using Linux with the Terminator shell. Maybe I could post what I have as a pull request, and you could check it out on your machine to see if it looks alright. It won't be a final solution but it would be a start.

**Comment by redbmk** [comment]
I totally misunderstood what you meant about 256 vs 16. I guess I was thinking 256 was better cause it was a higher number than 16. I switched [Terminator to using base16](https://github.com/joerx/base16-terminator) and while my terminal actually looks a lot nicer, toothpaste still doesn't look right.

**Comment by imcatnoone** [comment]
Thanks for tackling this @redbmk. You're awesome. Please let me or @CodeTheory know what we can do to help! Even if it comes down to QA to ensure it looks as close to Toothpaste as possible in order to solve the initial problem!

**Comment by redbmk** [comment]
I think it could probably use a little more work eventually but yeah I think if @jondot could take a look at it in his terminal that would be helpful. I only changed one color around, in vimdiff, but I'm pretty sure vimdiff is still all screwed up. All I really did was take out the background color for a few different scenarios - which means it would be black, white, or transparent depending on your terminal. Might end up looking weird with a white background.

If I get around to tweaking the vimdiff colors then def having one of you QA it would be helpful.

**Comment by imcatnoone** [comment]
@redbmk absolutely! Whether it is you or @jondot we'd love to help get you going and ensure Toothpaste looks fantastic and in turn, functions 100% for you. Always around for QA! Worse comes to worse, just ping me on[ twitter](http://twitter.com/imcatnoone).
